### PR TITLE
medical file api completed, patch and get

### PR DESCRIPTION
**Pull request by aarushXL**
created medical file routes for updating and getting

### CODE REVIEW

File: src/routers/subscriber.routes.js [hunk 1]
```diff
@@ -54,6 +57,15 @@ router.post(
 );
 router.patch("/:subscriberId", updateSubscriber);
 
+// ---------------------------------------- just medical files router -----------------------------------------------
+
+router
+.route(`/:subscriberId/beneficiaries/:beneficiaryId/medicalFiles`)
+.patch(customValidation(createSimpleMedicalFileValidator , "body") , createMedicalFilesController)
```
Comment (afourteia) [inline src/routers/subscriber.routes.js:64]: rename to updateSimpleMedicalFilevalidator for clarity

File: src/controllers/medicalFiles.controller.js [hunk 2]
```diff
@@ -232,7 +232,27 @@ const initMedicalFilesController = () => {
             return serverErrorResponse(res, error.message);
         }
     }
-
+    
+    const createMedicalFilesController = async (req, res) => {
```
Comment (afourteia) [inline src/controllers/medicalFiles.controller.js:236]: update to updateMedicalFileController for clarity

File: src/services/medicalFilesServices.js [hunk 3]
```diff
@@ -104,3 +105,25 @@ exports.createMedicalFiles = async (body, params, schema, fileData) => {
...
+      _id : user._id,
+      beneficiaryId : user?.beneficiaryId,
+      weight : user?.medicalFiles?.weight,
+      bloodType : user?.medicalFiles?.bloodType,
+      height : user?.medicalFiles?.height,
+    }
+  }
+  else{
+    return {}
+  }
```
Comment (afourteia) [inline src/services/medicalFilesServices.js:128]: No need for an extra service, just use the beneficiary services and then extract the data from it

**Review by afourteia — CHANGES_REQUESTED** [review]

File: src/controllers/medicalFiles.controller.js [hunk 4]
```diff
@@ -232,7 +232,27 @@ const initMedicalFilesController = () => {
...
+        try {
+            const query = {_id : req.params.beneficiaryId}
+            const createResponse = await medicalFilesServices.createMedFileForSubscriber(query , {medicalFiles : req.body} , "create");
```
[the remark this answers is not in the document]
Comment (aarushXL) [reply src/controllers/medicalFiles.controller.js:239]: beneficiary services does not exists, should I create a new file for it ?

**Review by afourteia — APPROVED** [review]

**Review by afourteia — APPROVED** [review]

**Pull request merged**